addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -430,3 +430,33 @@ test("show call participants in discuss sidebar", async () => {
         ],
     });
 });
+
+test("Sort call participants in side bar by name", async () => {
+    mockGetMedia();
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["discuss.channel.rtc.session"].create([
+        {
+            channel_member_id: pyEnv["discuss.channel.member"].create({
+                channel_id: channelId,
+                partner_id: pyEnv["res.partner"].create({ name: "AAA" }),
+            }),
+            channel_id: channelId,
+        },
+        {
+            channel_member_id: pyEnv["discuss.channel.member"].create({
+                channel_id: channelId,
+                partner_id: pyEnv["res.partner"].create({ name: "BBB" }),
+            }),
+            channel_id: channelId,
+        },
+    ]);
+    await start();
+    await openDiscuss(channelId);
+    await contains("div:nth-child(1) .o-mail-DiscussSidebarCallParticipants", {
+        contains: [".o-mail-DiscussSidebarCallParticipants-name:contains('AAA')"],
+    });
+    await contains("div:nth-child(2) .o-mail-DiscussSidebarCallParticipants", {
+        contains: [".o-mail-DiscussSidebarCallParticipants-name:contains('AAA')"],
+    });
```

Comment:
Shouldn't one of these be `BBB`?
There's probably something wrong with the test if it passes.